src/main/Security/Services/Change2FAService.java

```diff
@@ -27,7 +34,7 @@ public Change2FAService(MongoDatabase db, Logger logger, String username, Boolea
   @Override
   public Message executeAndGetResponse() {
     Objects.requireNonNull(db);
-    User user = UserDao.findOneUserOrNull(db, username);
+    User user = getUserService.getUser(username);
```

Comment:
like for example here, instead of using the getUserService, we can just call userDao.get() which will return the optional user